biogeochem/FatesPatchMod.F90

```diff
@@ -409,24 +551,14 @@ subroutine ZeroValues(this)
       this%canopy_layer_tlai(:)              = 0.0_r8
       this%total_tree_area                   = 0.0_r8  
       this%zstar                             = 0.0_r8
-      this%elai_profile(:,:,:)               = 0.0_r8
+      
       this%c_stomata                         = 0.0_r8 
       this%c_lblayer                         = 0.0_r8
-      this%psn_z(:,:,:)                      = 0.0_r8
-      this%nrmlzd_parprof_pft_dir_z(:,:,:,:) = 0.0_r8
-      this%nrmlzd_parprof_pft_dif_z(:,:,:,:) = 0.0_r8
 
+      
       ! RADIATION
       this%rad_error(:)                      = 0.0_r8
-      this%fabd_sun_z(:,:,:)                 = 0.0_r8 
-      this%fabd_sha_z(:,:,:)                 = 0.0_r8 
-      this%fabi_sun_z(:,:,:)                 = 0.0_r8 
-      this%fabi_sha_z(:,:,:)                 = 0.0_r8  
-      this%ed_parsun_z(:,:,:)                = 0.0_r8 
-      this%ed_parsha_z(:,:,:)                = 0.0_r8
-      this%ed_laisun_z(:,:,:)           = 0._r8
-      this%ed_laisha_z(:,:,:)           = 0._r8
```

Comment:
I noticed there were not moved into the `ZeroDynamics` routine, although they are included in the `NanDynamics` routine.  Is this zero'd downstream?

Reply:
yes, it is zero'd here. 

https://github.com/NGEET/fates/blob/256dd624360598c25163f93da8b1767137a47cd4/radiation/FatesRadiationDriveMod.F90#L297-L300


I must had identified this as redundant.